styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -143,4 +147,28 @@ private int flyteErrorCodeToStyx(final String errorCode) {
           return UNKNOWN_ERROR_EXIT_CODE;
     }
   }
+
+  private ExecutionMode toFlyteExecutionMode(Trigger trigger) {
```

Comment:
should we unit test this function too?

Reply:
We are testing this through the unit test `testStyxTriggerTranslateToFlyteExecutionMode`.